tests/Test/Disciplina/Educator/Validate.hs

```diff
@@ -0,0 +1,95 @@
+module Test.Disciplina.Educator.Validate where
+
+import Test.Common
+
+import Disciplina.Core (ATGDelta (..), CourseId (..), Grade (..))
+import Disciplina.Crypto (PublicKey, SecretKey, hash, getMerkleRoot, fromFoldable, sign)
+import Disciplina.Educator ()
+
+import qualified Data.Map.Strict as M
+
+courseCompScience1 :: CourseId
+courseCompScience1 = CourseId 3
+
+studentAPubKey, studentBPubKey,
+  educatorJPubKey, educatorKPubKey :: PublicKey
+studentAPubKey = mkPubKey 'a'
+studentBPubKey = mkPubKey 'b'
+educatorJPubKey = mkPubKey 'j'
+educatorKPubKey = mkPubKey 'k'
```

Comment:
You can just generate a bunch of keypairs randomly in advance, using e. g. `unsafePerformIO`.

Reply:
I added txs generated from unsafePerformIO some lines below. I rather keep these to have control over some of the test keys.